napari_skimage_regionprops/_regionprops.py

```diff
@@ -184,3 +196,371 @@ def ellipsoid_axis_lengths(table):
+                    ref_channel_props[props].tolist())
+
+                label_links[props + suffixes[ref_channel]] = \
+                    label_links['label' + suffixes[ref_channel]]
+
+                label_links = label_links.replace(
+                    {props + suffixes[ref_channel]: props_mapping}
+                    )
+
+                if large_numbers:
+                    label_links[props + suffixes[ref_channel]] = \
+                        label_links[props + suffixes[ref_channel]]\
+                        .astype(props_dtype)
+
+            col_names = label_links.columns.to_list()
+            
+            # Append properties of probe channel to table
+            if intensity_image is not None:
+                # If single intensity image was given, then use it
+                if intensity_image.shape[0]==1:
+                    image = intensity_image[0]
+                # Otherwise use the corresponding intensity image
+                else:
+                    image = intensity_image[i]
+            probe_channel_props = pd.DataFrame(
+                regionprops_table(image=image,
+                                  labels=labels_array[i],
+                                  **kwargs)
+            )
+            # rename columns
+            probe_channel_props.rename(
+                columns=dict([(props, props + suffixes[i])
+                              for props in properties_with_extras]),
+                inplace=True)
+            probe_channel_props.rename(columns={'label': 'label' + suffixes[i]},
+                                       inplace=True)
+            
+            # Merge reference table *label_links) with probe table
+            # (probe_channel_props) based on column "label-ch probe#" 
+            # 'outer' indicates to include everything and fill with nans
+            table = label_links.merge(probe_channel_props,
+                                      on = 'label' + suffixes[i],
+                                      how = 'outer')
+
+            # Insert new column names (from probe channel), except 'label-ch #'
+            probe_column_names = probe_channel_props.columns.to_list()[1:]
+            col_names = col_names[1:] + [col_names[0]] + probe_column_names
+            # Re-order columns
+            table = table[col_names]
+            # Re-order rows
+            table = table.sort_values(by=['label' + suffixes[ref_channel]]) \
+                .reset_index().iloc[:,1:]
+            if summary:
+                grouped = table.groupby('label' + suffixes[ref_channel])
+                table = grouped[probe_column_names].describe().reset_index()
+            table_list += [table]
+
+    return table_list
+
+
+regionprops_map_channels_table_all_frames = analyze_all_frames(
+    napari_regionprops_map_channels_table)
+register_function(
+    regionprops_map_channels_table_all_frames,
+    menu="Measurement > Regionprops map multichannel of all frames (nsr)")
```

Comment:
Might there be a more human readable title such as `Measure multiple images in th context of one label image` or something similar?